packages/docs/docs/guides/observability.md

```diff
@@ -180,6 +180,21 @@ function MyTracer(props: { children: AI.Node }, { wrapRender }: AI.ComponentCont
 
 This technique uses the [context affordance](./rules-of-jsx.md#context).
 
+### OpenTelemetry Integration
```

Comment:
Should we add a more explicit note about DataDog?

Reply:
Added a link to the OTel vendors page, which includes Datadog: https://opentelemetry.io/ecosystem/vendors/

Feel free to edit further as you like :)